modules/subscriptions.php

```diff
@@ -525,6 +525,7 @@ function get_settings() {
 	 *	not_subscribed  : strange error.  Jetpack servers at WordPress.com could subscribe the email.
 	 *	disabled        : Site owner has disabled subscriptions.
 	 *	active          : Already subscribed.
+	 *  pending         : Tried to subscribe before but the confirmation link is never clicked. No confirmation email is sent.
```

Comment:
Hmm... don't know why this indentation exists. Will add another commit to fix this.